doc(release): add release note for centreon-autodiscovery-21.04.3 (#1164)
centreon/centreon-documentation: en/releases/centreon-commercial-extensions.md

## Centreon Auto Discovery

### 21.04.3

> Release date: `null`

Reviewer: ```suggestion
Release date: `November 24, 2021`
```

#### Improvements

- Improved the way search requests are sent to the backend. The old way caused errors of type 500 when all search filters were removed.

#### Bug fixes

> - Fixed invalid upgrade.php file that caused errors in the update process and could cause the loss of existing jobs

Author: ```suggestion
- An invalid path in upgrade.php caused PHP errors in the update process
```
